src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
Yeah, it'd be good to check that app. @bshyong might know the best way to log in and see.